### DISCUSSION TITLE
Raising more awareness of this great tool

### DISCUSSION THREAD

**Discussion by sheldonhull** [aquaproj/aqua#1650] [Deprecated (Proposal)]
Let's brainstorm. 

This is one of the best tools for setting up my CI & local environment to be similar and far better than a bunch of custom bash scripts.

Because it offers version pinning it provides better control in CI than using brew and is far faster typically. 

I want others to be aware of it.

A few ideas:

- [ ] Join the discord gophers group and get a channel for the project to raise visibility. https://github.com/aquaproj/aqua/issues/1660
- [ ] Highlight it as an alternative to brew for cask like tools that offer better version control. https://github.com/aquaproj/aquaproj.github.io/issues/268
- [ ] More quick start with devcontainers showing up as an available feature. https://github.com/aquaproj/aqua/issues/1659

**Comment by suzuki-shunsuke** [comment]
Thank you for your proposal.
I'm really glad that you like aqua.

## Join the discord gophers group and get a channel for the project to raise visibility.

This is a good idea to make aqua more popular, but I have a concern.

Ideally, I'd like to keep discussions in one place (GitHub) so that users and maintainers can find the information and track the topic easily.
If the information is dispersed, users can't find the information and track the topic.

When I checked an issue of a project, I was dissappointed because they talk about the issue in discord and I couldn't catch up the status of the issue from the issue. I saw discord, but in the channel various topics are discussed and I couldn't find the information about the issue.

I'd like to avoid the same situation.

And it isn't good to discuss the same topic in the multiple places.

Chat is a flow style communication, so it is difficult to stock the information.
And we have to answer the same question for many times.

Users can post comments roughly. This is good for users but isn't good for maintainers.
Chat has no template unlike GitHub Issue, so user's question would lack required information so we have to ask users question to get required information. It's very tough work.

I hope the community will grow and users support each other without me, but it would not so easy.

---

I described my concern, but I have no idea to make aqua more popular and introducing discord seems good to me.

## Highlight it as an alternative to brew for cask like tools that offer better version control.

We have already described the attractive of aqua, but we could emphasize and clarify them more and more.

- https://aquaproj.github.io/docs/overview#why-aqua
- https://aquaproj.github.io/docs/comparison/common
- https://aquaproj.github.io/docs/comparison/homebrew

## More quick start with devcontainers showing up as an available feature.

Related discussion: https://github.com/orgs/aquaproj/discussions/1273

Sorry but I'm not familiar with devcontainers.
I learnt devcontainers before, but I forgot. 😅

**Reply by suzuki-shunsuke** [reply]
I'm not familiar with Discord, but I joined gopher group from https://discord.com/invite/golang .

**Comment by suzuki-shunsuke** [comment]
I created some GitHub Issues and arranged GitHub Project.
There are many issues and some of them are tough work, but we will handle them one by one.

https://github.com/orgs/aquaproj/projects/8

**Comment by suzuki-shunsuke** [comment]
Moved this discussion to https://github.com/aquaproj/aqua/issues/1671

Background: https://github.com/aquaproj/aqua/issues/1666
